docs/UserGuide.md

```diff
@@ -107,6 +109,14 @@ Adds a person to UniGenda without needing complete information about the person.
 
 Format: `add n/NAME p/PHONE_NUMBER [tg/TELEGRAM] [gh/GITHUB] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
 * Duplicates of (Name, Phone Number) contacts will be detected; you cannot have two people with the same combination of (Name, Phone Number).
+* `NAME` should only contain alphanumeric characters, not blank, and at most 20 characters long.
+* `PHONE_NUMBER` should only contain numeric characters, at least 3 digits long, and at most 15 digits long.
+* `TELEGRAM` should only contain alphanumeric characters and underscore (_), at least 5 characters long, and not exceeding 40 characters.
+* `GITHUB` should only contain alphanumeric characters and hyphen (-), doesn't have multiple consecutive hyphens or begin or end with a hyphen, not blank, and at most 39 characters.
+* `ADDRESS` should only contain alphanumeric characters and the following punctuations: **!"#$&'()\*+,-.:;<=>?@**. Its length should not exceed 80 characters.
+* `EMAIL` should be a **valid** email with length not exceeding 60 characters.
```

Comment:
I think it might be misleading to say valid email, because people can interpret it as the email must exist.